js/interface.js

```diff
@@ -2,6 +2,7 @@ var $initialSpinnerLoading = $('.spinner-holder');
 var $contents = $('#contents');
 var $sourceContents = $('#source-contents');
 var $dataSources = $('#data-sources > tbody');
+var $trashSources = $('#trash-sources > tbody');
```

Comment:
"Trash sources" seem like a mistranslation or a different context, please refer to them as "Trashed Data Sources"